crates/burn-tensor/src/tests/mod.rs

```diff
@@ -107,6 +107,7 @@ macro_rules! testgen_quantization {
         burn_tensor::testgen_q_sin!();
         burn_tensor::testgen_q_slice!();
         burn_tensor::testgen_q_sort_argsort!();
+        // burn_tensor::testgen_q_split!();
```

Comment:
@antimora @louisfd It looks like there's some sort of nuance here that I'm a bit out of the loop on- do I need to do something extra for the quantized set of tests/do I even need quantized tests here?

Reply:
Adding quantized tests is not well documented at the moment, so you're not expected to do it. 
@laggui will work on making it more straightforward and he'll be able to adapt your non-quantized tests.